src/main/java/com/gooddata/executeafm/ObjIdentifierQualifier.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (C) 2017, GoodData(R) Corporation. All rights reserved.
+ */
+package com.gooddata.executeafm;
+
+import com.fasterxml.jackson.annotation.JsonRootName;
+import com.fasterxml.jackson.annotation.JsonValue;
+import com.gooddata.util.GoodDataToStringBuilder;
+
+import java.util.Objects;
+
+/**
+ * Qualifies metadata {@link com.gooddata.md.Obj} using an identifier
+ */
+@JsonRootName("identifier")
+public final class ObjIdentifierQualifier implements ObjQualifier {
```

Comment:
Why not `IdentifierObjQualifier`?

Reply:
to me the former is easier to pronounce, otherwise I don't care... both names are perfectly valid to me.

Reply:
In addition we want to be consistent with resource specification and there iis `ObjIdentifierQualifier` but we can decide to change even that in this phase, @martiner do you have some preference?